guides/personal_mixed_skin.md

```diff
@@ -0,0 +1,56 @@
+- .png and .jpg - osu! **only** supports .png images for skin elements, with **one** exception - `menu-background.jpg`, which can only be a .jpg image.
+- .mp3 and .wav - While osu! supports both .mp3 and .wav sounds, it prioritizes .wav sounds. If `failsound.mp3` and `failsound.wav` are both present, `failsound.wav` will play when you fail a map. As .mp3 is much smaller than .wav, it is recommended to use .mp3 for longer sounds such as `failsound.mp3` and `applause.mp3`. It is not recommended to use .mp3 for gameplay or interface sounds as they have a slight delay when playing.
+- `skin.ini` - This file defines skin behavior, such as whether the cursor should expand or not, what color your combo colors are, etc. There are many more commands that dictate your skin's properties so read the `The skin.ini` section of this guide for more information.
+- All animation frames start at 0.
+
+## Mixing images
+
+Mixing images is simple if you learn the different file names, or consult the skinnable files list. The steps are as follows:
+
+1. Open the folders of the skins you want to mix from in your file explorer. These folders can be found by navigating to the `Skins\` directory of your osu! installation. By default this will be located at `C:\Users\[User Name]\AppData\Local\osu!\Skins\`. You can locate this by pressing `CTRL + O` in the osu! window to bring up settings, clicking `Open osu! folder`, and navigating to the `Skins\` directory. Alternatively, you can go to settings, navigate to the `SKIN` section, and click `Open current skin folder` for all skins you want to mix.
+2. Create a new folder in the `Skins\` directory and name it something that you will remember, we will call our example `mixedskin`.
+3. Decide what images from which skins you want to use. For example, we want to use the cursor from `skin1`, the hitcircle and hitcircle overlay from `skin2`, and so on, into `mixedskin`.
+4. Move these images into the folder we created. We move `cursor.png` and `cursortrail.png` from `skin1`, and `hitcircle.png` and `hitcircleoverlay.png` from `skin2`, and all of their HD equivalents, if present, into `mixedskin`
+5. Reload your skins with `CTRL + ALT + SHIFT + S` , and select the mix skin from the drop down list. With our example, it should be the default skin with `skin1`'s cursor and `skin2`'s hitcircle. Make sure to reload your skin after all changes you make for those changes to take effect.

```

Comment:
There is no need to reload the skin when you just changed to it. Feel like this paragraph overall needs heavy restructuring, but dont have a suggestion right now

Reply:
i have no idea what i was writing here
added a step 3. to select the skin and adjusted (now) step 6. accordingly